src/frontend/components/registrationForm.js

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from "react";
+import Input from "./input";
+import Button from "./button";
+import Select from "./Select";
+
+class RegistrationForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+  handleChange = () => {
+    console.log("Changes made in input ");
+  };
+  getDays = () => {
+    const days = ["Day"];
+    let newDate = new Date();
+    let date = newDate.getDate();
+    Array.from(new Array(31), (v, i) => (
+      days.push(date)
+    ));
+    return days
+  };
+  getMonths = () => {
+    var months = [
+      "month",
+      "Jan",
+      "Feb",
+      "Mar",
+      "Apr",
+      "May",
+      "Jun",
+      "Jul",
+      "Aug",
+      "Sep",
+      "Oct",
+      "Nov",
+      "Dec"
+    ];
+    return  months
+  };
+  getYears = () => {
```

Comment:
You could use a similar approach as I stated under the `getDays` call. Also, if you want to learn, you can use a timer to benchmark the efficiency of my approach vs yours.